tests/functional/api/groups/test_members.py

```diff
@@ -265,18 +183,3 @@ def group_member_with_token(group_member, db_session, factories):
     db_session.add(token)
     db_session.commit()
     return (group_member, token)
-
-
-@pytest.fixture
-def user_with_token(db_session, factories):
-    user = factories.User()
-    token = factories.DeveloperToken(userid=user.userid)
-    db_session.add(token)
-    db_session.commit()
-    return (user, token)
```

Comment:
The `user_with_token` is not the `user` here (but is in other places). The token is never actually used from `user_with_token`. In the end it was completely replaced by just passing user to `token_auth_header` below.